app/Http/Middleware/AcitivityLog.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use App\Activity;
+use Closure;
+
+class AcitivityLog
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @return mixed
+     */
+    public function handle($request, Closure $next)
+    {
+
+        $response = $next($request);
+
+        if(auth()->user()) {
+
+            #   Change description according to and from route.
+
+            $description = "";
+
+            if(request()->path() == "login" ){
```

Comment:
The structure of the code must follow this style:

if (condition) {
    do something
}
else if (condition) {
    do something
}
else {
    do something
}

Reply:
Restructured to the format above.